rules/config

```diff
@@ -294,3 +294,6 @@ SONIC_SLAVE_DOCKER_DRIVER ?= vfs
 # GZ_COMPRESS_PROGRAM - select pigz (a parallel implementation of gzip) to reduce a build time
 # and speed up a decompression of docker images on target system
 GZ_COMPRESS_PROGRAM ?= gzip
+
+# SONIC_OS_VERSION - sonic os version
+SONIC_OS_VERSION ?= 11
```

Comment:
Does this 11 correspond to the Debian version? If so, wouldn't it be better to read from `/etc/lsb_release.yml` and parse the version field there, rather than having it hardcoded here?